python/Ganga/Core/GangaThread/WorkerThreads/WorkerThreadPool.py

```diff
@@ -222,6 +222,14 @@ def worker_status(self):
         """
         return [(w.gangaName, w._command, w._timeout) for w in self.__worker_threads]
 
+    def threads_matching(self, name_str):
```

Comment:
I plan to use this in the future in a few places and was experimenting with it in this branch